src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/dashboardnotifications/claimant/OrderMadeClaimantNotificationHandler.java

```diff
@@ -110,6 +115,9 @@ && hasTrackChanged(caseData)) {
         if (isSDODrawnPreCPRelease()) {
             return SCENARIO_AAA6_CLAIMANT_SDO_DRAWN_PRE_CASE_PROGRESSION.getScenario();
         }
+        if (isFinalOrderIssued(callbackParams)) {
+            return SCENARIO_AAA6_UPDATE_DASHBOARD_CLAIMANT_TASK_LIST_UPLOAD_DOCUMENTS_FINAL_ORDERS.getScenario();
```

Comment:
I think this should still trigger the same notification as SCENARIO_AAA6_CP_ORDER_MADE_CLAIMANT - try to add that to the SQL.